scene/resources/audio_stream_polyphonic.cpp

```diff
@@ -34,6 +34,9 @@
 #include "scene/main/scene_tree.h"
 #include "servers/audio_server.h"
 
+constexpr uint64_t ID_MASK = 0x0FFFFFFFF;
```

Comment:
I first thought you had changed the mask with that extra leading 0, as I thought it was `0x0FFF_FFFF` before couting the `F`s.

Makes more sense to me to only include 8 hex and not 9, so our brains can safely assume that a long-ish `0xFF...FFF` might indeed be the common `0xFFFF_FFFF`.
```suggestion
constexpr uint64_t ID_MASK = 0xFFFF_FFFF;
```

Reply:
Amended myself.